src/components/ChallengeModal/ChallengeModal.tsx

```diff
@@ -1948,10 +1971,11 @@ export function challenge_text_description(challenge) {
         (g.disable_analysis ? ", " + _("analysis disabled") : "");
     if (c.challenger_color !== "automatic") {
         let yourcolor = "";
+
         if (
             data.get("user") &&
-            ((c.challenger && c.challenger.id !== data.get("user").id) ||
-                (c.user && c.user.id !== data.get("user").id))
+            ((c as any)?.challenger.id !== data.get("user").id ||
+                (c as any)?.user.id !== data.get("user").id)
```

Comment:
Should include `?` after `user` - this member is not set in in `ChallengesList` (me/challenges). Example challenge:

```
{
		"id": 7065,
		"challenger": {
			"id": 1300,
			...
		},
		"challenged": {
			"id": 1371,
			...
		},
		"game": { ... }
		},
		"group": null,
		"challenger_color": "black",
		"aga_rated": false
	}
```

Reply:
Good catch, thanks!